docs/source/methods/distillation.md

```diff
@@ -40,3 +40,8 @@ The following are the default augmentation settings for Distillation. To learn h
 
 ```{include} _auto/distillation_transform_args.md
 ```
+
+## Note
+
+Starting from **LightlyTrain 0.7**, `method="distillation"` uses a new, improved `v2` implementation.\
+The previous version is still available via `method="distillationv1"` for backward compatibility.
```

Comment:
````suggestion
```{note}
Starting from **LightlyTrain 0.7.0***, `method="distillation"` uses a new, improved `v2` implementation
that achieves higher accuracy and trains up to 3x faster. The previous version is still available via
`method="distillationv1"` for backward compatibility.
```
````

Lets put this at the top right after the `# Distillation (recommended)` intro paragraph.